languages/swift/exercises/concept/string-components/.docs/introduction.md

```diff
@@ -0,0 +1,62 @@
+We've seen how to build up strings in previous exercises. Here we will see some of the ways to break strings apart.
```

Comment:
I think the introduction might be trying to do a bit too much explaining. Ideally, the introduction should be relatively short and concise. Quoting [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd):

> Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the after.md.

This introduction also has a lot of code samples, but I'm unsure if they should be part of the introduction.md as to me it looks like they don't introduce new syntax. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd):

> Code examples should only be used to introduce new syntax 

Of course, the after.md can include these examples.

Maybe you could remove the code examples that don't introduce new syntax and try to shorten the introduction text a bit?

Reply:
I like the new version!